sematic/ui/packages/common/src/component/RunReferenceLink.tsx

```diff
@@ -0,0 +1,16 @@
+import Link from "@mui/material/Link";
+
+interface RunReferenceLinkProps {
+    runId: string;
+    className?: string;
+}
+
+const RunReferenceLink = (props: RunReferenceLinkProps) => {
+    const { runId, className } = props;
+
+    return <Link href={`/runs/${runId}`} variant={"small"} type={"code"} className={className}>
```

Comment:
The `"code"` type and the `Typography` element this is being wrapped in [here](https://github.com/sematic-ai/sematic/pull/708/files#diff-14b6c607c7a0cadba723220c43a0e36824627f7132eee953654c13bb89ea89d2R106) make it seem that this isn't a link. I only discovered that it is by mousing over it.

In the Git information panel, there are also links that use grey Typography, and they are also not obvious as links, but they do have a different font when compared to non-link text right next to them.

Also, the current Notes run id has a copy icon, and the current and new Git info links have copy icons, but the new Notes run id does not.

These inconsistencies make it unclear to the user which bit of text is a link and which is not. Please standardize by coloring all links in blue, not just on mouse-over, and provide copy buttons as well.

cc @neutralino1 

New Notes run ids:
<img width="155" alt="image" src="https://user-images.githubusercontent.com/1894533/229593154-5104ee54-0be2-4a7f-ba77-50113f20e99d.png">

<img width="166" alt="image" src="https://user-images.githubusercontent.com/1894533/229593195-ddefa457-ab8b-4cc7-be29-6ffdec7a5b57.png">

New Git info panels:
<img width="209" alt="image" src="https://user-images.githubusercontent.com/1894533/229593488-a193cfa1-d6aa-417b-9a4e-e3f7d3372330.png">

<img width="208" alt="image" src="https://user-images.githubusercontent.com/1894533/229593584-b9ef402f-65c9-4b1a-9a52-f0792acb911e.png">

Current Notes run id:
<img width="209" alt="image" src="https://user-images.githubusercontent.com/1894533/229593970-17a5c647-9afb-4c53-a43a-ed27a4b8457f.png">

Reply:
Thanks, @tscurtu . What you proposed makes sense. I will make some changes.

The font for the `pf49df3` part on the Git panel is wrong, I will fix that. It is supposed to be monospaced font like `<RunReferenceLink />`

@neutralino1 By what I can see from the Figma design, maybe we are intentionally weakening the styles of links, is it the case?

Reply:
I am fine with only turning blue on hover as long as all links have a specific style that isn't used by any other text.